app/templates/cookies.html

```diff
@@ -66,7 +66,7 @@ <h3>{{ _('Cookies that help with our communications') }}</h3>
             {{ onsRadios({
                 "id": "cookies-measure-website-comms",
                 "name": "cookies-campaigns",
-                "legend": _("Do you want to allow communications tracking?"),
+                "legend": "Do you want to allow communications tracking?",
```

Comment:
This one is tricky, it's not visible, but it is still rendered on the page and maybe still needs to be translated? Screen readers might still pick this up. The radio button example on the design pattern page doesn't even have a `legend` field, but it doesn't appear to work without it 🤔 

I think this legend still needs to be translated, but possibly it can just be the "question" title repeated instead of another different bit of text.

Reply:
Yeah, it's a weird one, even more so that the HTML [<legend>](https://www.w3schools.com/tags/tag_legend.asp) tag normally is displayed on the page. Might be worth letting know the design system team about it.

But for now it's a good shout what Adam said, we could just copy over the text from h3 headers for the sections containing the radios.

Reply:
It's weird, if I search for that text on the page in Firefox, it reports 1 match, but the match is invisible. 

The design we have doesn't appear to feature the text "`Do you want to allow communications tracking?`", so I think the best bet would be to re-use the already translated h3 title for the section as the legend for now.

Reply:
I think it's the class we're applying to it that's hiding it, so might be intentional on the part of whoever originally put that page together.